ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -61,7 +61,7 @@ public final void setup() throws Exception {
 
 		resetPropertiesFile = System
 				.getProperty("com.arjuna.ats.arjuna.common.propertiesFile");
-		if (resetPropertiesFile == null) {
+		if (resetPropertiesFile == null || !resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml")) {
```

Comment:
So that means in the tearDown() it should just clear this property 
`System.clearProperty("com.arjuna.ats.arjuna.common.propertiesFile");`